### PR TITLE
Use one main function.

### PR DESCRIPTION
**Pull request by XhmikosR**
Non-whitespace diff: <https://github.com/addyosmani/oust/pull/31/files?w=1>

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -47,20 +47,7 @@ const types = {
...
-    return $(chosenType.selector).map((i, el) => {
-        return $(el).attr(chosenType.attribute);
-    }).toArray();
-};
-
-module.exports.raw = (src, type) => {
+function oust(src, type, raw) {
     if (!src || !type) {
         throw new Error('`src` and `type` required');
     }
```
Comment (bezoerb) [inline index.js]: Maybe add another check for `chosenType` so we don't end up with `Cannot read property 'attribute' of undefined`

Comment (XhmikosR) [reply index.js]: But `chosenType = types[type]`, the same error happens in master too :)

Feel free to push here and I will check out your solution.

Comment (bezoerb) [reply index.js]: Doesn't mean we have to keep it that way ;)

**Comment by XhmikosR** [comment]
@bezoerb should we release a 0.6.0 or 1.0.0? I don't think with the current changes so far the next release can be a patch release.

https://github.com/addyosmani/oust/compare/v0.5.2...master

**Comment by bezoerb** [comment]
maybe 1.0 so we're finally `stable` ;)
But i would also be fine with `v0.6.0`

**Comment by XhmikosR** [comment]
Alright, let's go with v1.0.0 then :)

**Pull request merged**